<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RepresentationUI</class>
 <widget class="QMainWindow" name="RepresentationUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>658</width>
    <height>627</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>DREAM.3D</string>
  </property>
  <widget class="QWidget" name="centerWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <property name="spacing">
     <number>4</number>
    </property>
    <property name="leftMargin">
     <number>4</number>
    </property>
    <property name="topMargin">
     <number>4</number>
    </property>
    <property name="rightMargin">
     <number>4</number>
    </property>
    <property name="bottomMargin">
     <number>6</number>
    </property>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>658</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menu_RecentFiles">
     <property name="title">
      <string>Recent Files</string>
     </property>
     <addaction name="actionSomething"/>
    </widget>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionClose"/>
    <addaction name="separator"/>
    <addaction name="menu_RecentFiles"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuPlugins">
    <property name="title">
     <string>Plugins</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuPlugins"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionSomething">
   <property name="text">
    <string>Something</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../../../../Resources/icons/images/Icons.qrc"/>
  <include location="../../../../Resources/icons/images/Icons.qrc"/>
  <include location="../../../../Resources/icons/images/Icons.qrc"/>
  <include location="../../../../Resources/icons/images/Icons.qrc"/>
  <include location="../../../../Resources/icons/images/Icons.qrc"/>
  <include location="../../../../Resources/icons/images/Icons.qrc"/>
  <include location="../../../../Resources/icons/images/Icons.qrc"/>
  <include location="../../../../Resources/icons/images/Icons.qrc"/>
  <include location="../../../../Resources/icons/images/Icons.qrc"/>
  <include location="../../../../Resources/icons/images/Icons.qrc"/>
  <include location="../../../../Resources/icons/images/Icons.qrc"/>
  <include location="../../../../Resources/icons/images/Icons.qrc"/>
  <include location="../../../../Resources/icons/images/Icons.qrc"/>
  <include location="../../../../Resources/icons/images/Icons.qrc"/>
  <include location="../../../../Resources/icons/images/Icons.qrc"/>
  <include location="../../../../Resources/icons/images/Icons.qrc"/>
  <include location="../../../../Resources/icons/images/Icons.qrc"/>
  <include location="../../../../Resources/icons/images/Icons.qrc"/>
  <include location="../../../../Resources/icons/images/Icons.qrc"/>
  <include location="../../../../Resources/icons/images/Icons.qrc"/>
  <include location="../../../../Resources/icons/images/Icons.qrc"/>
  <include location="../../../../Resources/icons/images/Icons.qrc"/>
  <include location="../../../../Resources/icons/images/Icons.qrc"/>
  <include location="../../../../Resources/icons/images/Icons.qrc"/>
  <include location="../../../../Resources/icons/images/Icons.qrc"/>
  <include location="../../../../Resources/icons/images/Icons.qrc"/>
  <include location="../../../../Resources/icons/images/Icons.qrc"/>
  <include location="../../../../Resources/icons/images/Icons.qrc"/>
  <include location="../../../../Resources/icons/images/Icons.qrc"/>
  <include location="../../../../Resources/icons/images/Icons.qrc"/>
  <include location="../../../../Resources/icons/images/Icons.qrc"/>
  <include location="../../../../Resources/icons/images/Icons.qrc"/>
  <include location="../../../../Resources/icons/images/Icons.qrc"/>
  <include location="../../../../Resources/icons/images/Icons.qrc"/>
  <include location="../../../../Resources/icons/images/Icons.qrc"/>
  <include location="../../../../Resources/icons/images/Icons.qrc"/>
  <include location="../../../../Resources/icons/images/Icons.qrc"/>
  <include location="../../../../Resources/icons/images/Icons.qrc"/>
  <include location="../../../../Resources/icons/images/Icons.qrc"/>
  <include location="../../../../Resources/icons/images/Icons.qrc"/>
  <include location="../../../../Resources/icons/images/Icons.qrc"/>
  <include location="../../../../Resources/icons/images/Icons.qrc"/>
  <include location="../../../../Resources/icons/images/Icons.qrc"/>
  <include location="../../../../Resources/icons/images/Icons.qrc"/>
  <include location="../../../../Resources/icons/images/Icons.qrc"/>
  <include location="../../../../Resources/icons/images/Icons.qrc"/>
  <include location="../../../../Resources/icons/images/Icons.qrc"/>
 </resources>
 <connections/>
</ui>
